r.css

```
@@ -0,0 +1,240 @@
+        
+*{
+    padding: 0;
+    margin: 0;
+    
+}
+a{
+    text-decoration: none;
+    color:black;
+}
+header a:hover{
+    color:white;
```

It's better to set space everywhere after `":"`

```suggestion
    color: white;
```